source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+
+After enabling site replication, identity and access management (IAM) settings sync in the following order:
+
+#. Policies
+#. User accounts (for local users)
+   
+   Note: Sites that use OIDC and LDAP don't sync, and they may not have local users.
+
+#. Groups 
+   
+   For MinIO IDP and LDAP managed groups
+
+#. Service accounts
+
+   Only static accounts with valid claims.
+   Service accounts for ``root`` don't sync.
+
+#. Policy mapping for synced user accounts
+
+#. Policy mapping for `Short Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+Site healing
+------------
+
+With site replication enabled, bucket metadata and IAM entries can heal from whichever peer site has the most updated entry.
+
+
+Prerequisites
+-------------
+
+- Only *one* site can have data at the time of setup.
+  The other sites must be empty of buckets and objects.
+
+  After configuring site replication, the data replicates to the other sites.
+  Objects written to any site after setting up replication then replicate to all sites.
+
+- All sites must have the same deployment credentials (for example, ``MINIO_ROOT_USER``, ``MINIO_ROOT_PASSWORD``)
+- All sites must use the same external IDP(s), if any
+- For :ref:`SSE-S3 <minio-encryption-sse-s3>` or :ref:`SSE-KMS <minio-encryption-sse-kms>` encryption via KMS, all sites must have access to a central KMS deployment. 
+  This can be achieved via a central KES server or multiple KES servers (say one per site) connected via a central KMS (Vault) server.
+
+
+Configure site replication
+--------------------------
+
+.. tab-set::
+
+   .. tab-item:: Console
+
+      #. Deploy three or more separate MinIO sites, using the same Identity Provider for each site
+
+         Only one site can have any buckets or objects on it.
+         The other sites must be empty.
+
+      #. In a browser, access the Console for one of the sites
+
+         For example, ``https://<addressforsite>:9000``
+         
+         If one of the sites has content, log in to the Console for that site.
+
+      #. Select **Settings**, then **Site Replication**
+
+         .. image:: /images/minio-console/console-settings-site-replication.png
+            :width: 400px
+            :alt: MinIO Console menu with the Settings heading expanded to show Site Repilication
+            :align: center
+      
+      #. Select :guilabel:`Add Sites +``
+
+         .. image:: /images/minio-console/console-settings-site-replication-add.png
+            :width: 600px
+            :alt: MinIO Console's Add Sites for Replication screen
+            :align: center
+
+      #. Make the following entries:
+
+         :Access Key: `(required)` The user name to use for signing in to each site. Should be the same across all sites.
+
+         :Secret Key: `(required)` The password for the user name to use for signing in to each site. Should be the same across all sites.
+
+         :Site Name: An alias to use for the site name.
```

Comment:
Since we have some crossover of "alias" as something specific to `mc`, "A name or other identifying text to associate to the site" would be fine.